src/cts/test/array_dummy.tcl

```diff
@@ -0,0 +1,157 @@
+# Build an array of tiles with misc flops and hook up a clk net for CTS
+# This demonstrates the trouble with blockage unaware fixing.
```

Comment:
It would be better to factor out the common portion rather than copy it.  The comment needs updating.

Reply:
Updated comments for now.   Will revisit for further changes.